tests/integration/security/healthcheck/mtls_healthcheck_test.go

```diff
@@ -52,13 +53,15 @@ func setupConfig(cfg *istio.Config) {
 // Currently this test can only pass on Prow with a real GKE cluster, and fail
 // on CircleCI with a Minikube. For more details, see https://github.com/istio/istio/issues/12754.
 func TestMtlsHealthCheck(t *testing.T) {
-	ctx := framework.NewContext(t)
-	defer ctx.Done(t)
-	ctx.RequireOrSkip(t, environment.Kube)
+	framework.
+		NewTest(t).
+		RequiresEnvironment(environment.Kube).
+		Label(label.Presubmit).
+		Run(func(ctx framework.TestContext) {
 
-	ns := namespace.ClaimOrFail(t, ctx, "default")
-	_, err := deployment.New(ctx, deployment.Config{
-		Yaml: `apiVersion: "authentication.istio.io/v1alpha1"
+			ns := namespace.ClaimOrFail(t, ctx, "default")
```

Comment:
Isn't this, ReauireEnvironment thing already required in `TestMain`? Or the benfit is to customize per TestXXX as presubmit or post-submit?